src/components/Layout/Footer/Footer.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { FC } from "react";
+
+import "@mozilla-protocol/core/protocol/css/protocol.css";
+
+interface Link {
+  /*Displayed string represents the link*/
+  label: string;
+  /*URL to which the link redirects*/
+  url: string;
+}
+
+interface FooterLinksSection {
+  /*Array of links to be displayed below the 
+  footerLinkSectionName*/
+  links: Link[];
+  /*Name to be displayed at the top of the footer
+  column*/
+  footerLinksSectionName: string;
+}
+
+export interface FooterProps {
+  /*Array of footer columns, each with a name to be displayed at 
+  the top of the column, and an array of links to be displayed 
+  below the name*/
+  footerLinksSections: FooterLinksSection[];
```

Comment:
Could you change it to /**<comment>*/ so it shows up in the storybook docs panel?